pppd/pathnames.h

```diff
@@ -33,9 +33,9 @@
 #define _PATH_AUTHUP	 _ROOT_PATH "/etc/ppp/auth-up"
 #define _PATH_AUTHDOWN	 _ROOT_PATH "/etc/ppp/auth-down"
 #define _PATH_TTYOPT	 _ROOT_PATH "/etc/ppp/options."
-#define _PATH_CONNERRS	 _ROOT_PATH "/etc/ppp/connect-errors"
+#define _PATH_CONNERRS	 _ROOT_PATH "/var/log/ppp/connect-errors"
```

Comment:
Why not to add:
`#define _PATH_VARLOG 	"/var/log/ppp/"`
after
`#define _PATH_VARRUN 	"/var/run/ppp/"`